ingestion/src/metadata/ingestion/source/database/bigquery/metadata.py

```diff
@@ -327,12 +329,40 @@ def yield_database_schema(
                 )
         yield Either(right=database_schema_request_obj)
 
+    def get_table_obj(self, table_name: str):
+        schema_name = self.context.database_schema.name.__root__
+        database = self.context.database.name.__root__
+        return self.client.get_table(f"{database}.{schema_name}.{table_name}")
```

Comment:
We can use the fqn builder here

Reply:
table fqn won’t be needed there because the client needs project.schema.table and not dbservice.project.schema.table we won’t be able to use fqn.build

Reply:
used fqn._build as suggested, thanks